src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/AddSingleCaseToMultipleService.java

```diff
@@ -92,13 +90,21 @@ public void addSingleCaseToMultipleLogic(String userToken, CaseData caseData, St
 
     }
 
+    private String createMultipleReferenceLink(CaseData caseData)
+    {
```

Comment:
You should be consistent with formatting. This opening brace should be on the previous line
https://google.github.io/styleguide/javaguide.html#s4.1-braces

Reply:
Noted